### ISSUE TITLE
`securityinsight` version 2022-01-01-preview can't compile

### ISSUE THREAD

**Issue by magodo** [Azure/azure-sdk-for-go#17470]
The [`securityinsight`](https://github.com/Azure/azure-sdk-for-go/blob/35fb64f82ef3b3308f55b1da37c1fec36bdd4166/services/preview/securityinsight/mgmt/2022-01-01-preview/securityinsight) can't compile (introduced in [v62.3.0](https://github.com/Azure/azure-sdk-for-go/releases/tag/v62.3.0)):

```
# github.com/Azure/azure-sdk-for-go/services/preview/securityinsight/mgmt/2022-01-01-preview/securityinsight
vendor/github.com/Azure/azure-sdk-for-go/services/preview/securityinsight/mgmt/2022-01-01-preview/securityinsight/version.go:13:31: missing argument in conversion to Version
vendor/github.com/Azure/azure-sdk-for-go/services/preview/securityinsight/mgmt/2022-01-01-preview/securityinsight/version.go:17:6: Version redeclared in this block
        vendor/github.com/Azure/azure-sdk-for-go/services/preview/securityinsight/mgmt/2022-01-01-preview/securityinsight/enums.go:1922:6: other declaration of Version
```

**Comment by lirenhe** [comment]
@ArcturusZhang, could you take a look?

**Comment by ArcturusZhang** [comment]
Hi @magodo thank you for this issue.
I confirmed this as a bug in the track 1 Go code generator. I will fix this in the generator and release a new version of this package ASAP.

**Comment by matthchr** [comment]
Any update on this? It seems broken in v63.3.0 as well.

**Comment by Thz4UQgy** [comment]
Hi, could you give me a status update of this feature request? Kind regards

**Comment by ArcturusZhang** [comment]
> Any update on this? It seems broken in v63.3.0 as well.

Sorry for the delay of response - we are fixing this here: https://github.com/Azure/autorest.go/pull/802

**Comment by ArcturusZhang** [comment]
The corresponding fix has been merged. We will release a new version to apply this fix to the reported package.

**Comment by magodo** [comment]
@ArcturusZhang Isn't track1 stopping release for now?

**Comment by Alancere** [comment]
@magodo Fixed in https://github.com/Azure/azure-sdk-for-go/releases/tag/v66.0.0

**Comment by magodo** [comment]
@Alancere We are using the sdk for multiple resources, though the issue is fixed in v66.0.0,  as the same commit removed a lot of packages that are under used by us, it means we can't upgrade the SDK to this version.

**Comment by tadelesh** [comment]
@magodo Do you mean some removed preview version in v66 are used (v65 also removed some, the package size is over limit, all removed version seems to have newer stable version)? As we are trying to increase the adoption rate for new version, we are trying to limit the release of the old version, only for deprecation/severe security issue.
